tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -2053,6 +2053,7 @@ public function checkElementList(
 			$responseXmlObject = $this->listFolderAndReturnResponseXml(
 				$user,
 				"/",
+				"infinity"
```

Comment:
hmm, let's first find out why using `$this->listFolderAndReturnResponseXml()` with only 2 parameters doesn't complain.
Also, we should use either `o` or `1` not `infinity`

Reply:
this might be the reason we are not seeing the error because depth:infinity is disabled by default in ocis
```php
if ($this->davPropfindDepthInfinityIsEnabled()) {
...
```